help/include/sidebar_index.md

```diff
@@ -147,7 +147,7 @@
 * [Pin a channel](/help/pin-a-channel)
 * [Change the color of a channel](/help/change-the-color-of-a-channel)
 * [Unsubscribe from a channel](/help/unsubscribe-from-a-channel)
-* [Manage inactive channels](/help/manage-inactive-channels)
+* [Show or hide inactive channels](/help/manage-inactive-channels)
```

Comment:
I worry this makes it sound a bit like like this describes is a manual step you can do -- like there's a menu item on such channels to hide them or something.

"Configure hiding inactive channels" might be closer?

Reply:
I dunno, the page also documents the manual way to do that, and that wording is awkward. Could do:

> Automatically hide inactive channels

?

Reply:
@shubham-padia Do we need to do something for this link being renamed (same URL as before)?

Reply:
Nope, see https://github.com/zulip/zulip/pull/35504#discussion_r2238121142